include/isr_xc.h

```diff
@@ -75,7 +75,7 @@
 //==================================================================================================
 #if defined(__GNUC__)  &&  defined(__MSP430__)
     /* This is the MSPGCC compiler */
-#define _ISR(a,b)    void __attribute__((interrupt (a))) b(void)        // Edit by Alex Mykyta
+#define _ISR(a,b)    void __attribute__((interrupt (a##_VECTOR))) b(void)        // Edit by Alex Mykyta
```

Comment:
because different preprocessor have different behaviour on macro resolving strategy. on TI compiler, macro resolves agressively, so that
`_ISR(a,b) `
substitutes 'a' right into endpoint expression, that on a TI headers defined as '(<a number> * <vectorsize>)' (here <value> means a number).
therefore `__attribute__((interrupt (a)))` insteard of a vector name 'a'? takes expressio above. and compiler reports error, it want vector name, not expression.
Contrary `a##_VECTOR` - generates an token, that should be resolved on next preprocessor cycle. but, imaginaly, it leave this name as is to compiler and one accepts this token ok.